tests/features/joinup_seo/news.feature

```diff
@@ -0,0 +1,53 @@
+@api
+Feature: As an owner of the website
+  in order for my news to be better visible on the web
+  I need proper metatag to be encapsulated in the html code.
+
+  Scenario: Basic metatags are attached as json schema on the page.
+    Given collections:
+      | title                      | state     |
+      | Joinup SEO news collection | validated |
+    And users:
+      | Username          | E-mail                 | First name | Family name |
+      | Joinup SEO author | joinup.seo@example.com | Kurk       | Smith       |
+    And "news" content:
+      | title           | headline                    | logo     | body                    | created                         | publication date                | changed                         | state     | author            | collection                 |
+      | Joinup SEO news | Headline of Joinup SEO news | logo.png | Body of Joinup SEO news | Sun, 01 Dec 2019 13:00:00 +0100 | Wed, 25 Dec 2019 13:00:00 +0100 | Wed, 01 Jan 2020 13:00:00 +0100 | validated | Joinup SEO author | Joinup SEO news collection |
+
+    When I visit the "Joinup SEO news" news
+    Then the metatag json should be attached in the page
```

Comment:
Let's capitalize all instances of `JSON`. It is not a regular word but an abbreviation of JavaScript Object Notation.